src/server.js

```diff
@@ -65,6 +56,7 @@ const migrate = async (isApi) => {
  * @returns {Promise<Server>} Returns the Server object
  */
 const createServer = async (port, isApi) => {
```

Comment:
Minor nitpick: the `isApi` is a bit of an anti-pattern.

Perhaps it would be better for `createServer()` to take an `api` parameter, which is already initialized. Or maybe even a path to the api we want to configure.

The same goes for the `await migrate(isApi)` and `createServer()` calls

Reply:
Hey thanks @lewisdaly I’ll look at making that change

Reply:
Hey thanks @lewisdaly I’ll look at making that change